tinlake-ui/utils/useWriteOffPercentage.ts

```diff
@@ -18,7 +18,8 @@ export async function calculateWriteOffPercentage(tinlake: ITinlake, loanId: num
     }
 
     return writeOffPercentage.div(new BN(10).pow(new BN(25))).toString()
-  } catch {
+  } catch (e) {
+    console.log(`Oops: ${e}`)
```

Comment:
Did you mean to leave this in?

Reply:
Lol yah I needed it for debugging, but saw that we have the same console.log in tinlake.js [here](https://github.com/centrifuge/tinlake-apps/blob/main/tinlake.js/src/actions/admin.ts#L223) so figured I'd leave it in.